src/plugins/data/common/search/aggs/metrics/median.ts

```diff
@@ -60,8 +60,9 @@ export const getMedianMetricAgg = () => {
         name: 'field',
         type: 'field',
         filterFieldTypes: [OSD_FIELD_TYPES.NUMBER, OSD_FIELD_TYPES.DATE, OSD_FIELD_TYPES.HISTOGRAM],
+      },
+      {
         write(agg, output) {
```

Comment:
`agg` is unused here now. Can be replaced with `_`

Reply:
will change